### PR TITLE
Conference-page-project  Desktop and mobile version Implementation 

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
This is the capstone project PR.
In this pull request, several changes were made, such as:

- All necessary files setup
- Added all necessary content to index.html
- Added all necessary content to about.html
- Added mobile CSS styles
- Added desktop CSS styles
- Created dynamic HTML page in JavaScript.
- Created dynamic Mobile Menu in JavaScrip.
- Updated Readme file.
- Fixed Linter Errors.
https://www.loom.com/share/d5f199304bf24e8291d892d8a3795851

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+## Authors
+
+- Cindy Shin
+  cc logo Attribution-NonCommercial 4.0 International (CC BY-NC 4.0)
+  👤 **Meqdam Al-qudah**
+
+- [GitHub](https://github.com/MeqdamAlqudah)
+- [Twitter](https://twitter.com/MeqdamQudah)
+- [LinkedIn](www.linkedin.com/in/meqdam-al-qudah-7514a21b5)
+=======
```
Comment (afanimartin) [inline README.md]: - Please add "Acknowledgements" section to give credit to the UI design.

File: dist/index .html [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+          </ul>
+        </div>
+      </header>
+
+      <div class="container">
+        <p class="display-1">"Hello! Sharing world"</p>
+        <h1 class="display-1 fw-bold">
+          Global Designers <br />summit<span class="display-1 fw-bold"
+            >2021</span
+          >
```
Comment (afanimartin) [inline dist/index .html]: ### original design
original-hero

### your design
review-hero

Please make sure your deign looks the same as the given design. Your navbar,the dates and the text need some work. The font size should be big enough like in the given mockup.

File: dist/index .html [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+          alt="fifith partner"
+        />
+      </div>
+      <div class="finalLogo">
+        <img
+          src="img/poth-version-img/FinalLogo.png"
+          class="finalLogoImage"
+          alt="final logo"
+        />
+        <p>2015 Creative Commons Korea Some Rights Reserved</p>
```
Comment (afanimartin) [inline dist/index .html:355]: ### original footer
original-footer
 Take note of the space between the logo and the text to the right, the two being centered. Also don't forget to give some margin. Please make the requested changes

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @MeqdamAlqudah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Css/queries.css [hunk 4]
```diff
@@ -1,5 +1,235 @@
...
+    display: flex;
+    margin: 0;
+    width: 49%;
+  }
+
+  .menuDesk li {
+    margin-left: 4ch;
+    margin-right: 1rem;
+    margin-bottom: 1rem;
+  }
```
Comment (VanessaAoki) [inline Css/queries.css]: - [ ] As requested by the previous reviewer, to make the header as similar to the Behance design, you could consider removing the `margin-bottom`, to make the elements horizontally centered.

File: Css/style.css [hunk 5]
```diff
@@ -500,30 +546,41 @@ footer.Partner h2::after {
...
+}
+
+.featuredSpeakers .speakers li .texContainer span:hover,
+.featuredSpeakers .speakers li .texContainer span:active {
+  text-decoration: underline;
 }
 
 .finalLogo p {
-  margin-top: 3rem;
+  margin-top: 1rem;
```
Comment (VanessaAoki) [inline Css/style.css:579]: - [ ] To match the original design better, you should make this horizontally centered.

Behance design:
image

Your design:
image

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @MeqdamAlqudah, 

Great work on making the changes requested by the previous reviewer, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


really close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -17,80 +17,129 @@
...
-          class="display-1 fw-bold"
-          >2021</span
-        >
-      </h1>
-      <div class="text-container">
-        <p>
-          A joufyl celebration believing in the value of opennes and sharing,
-          creating a positive change with people from over 80 countries is
-          taking place in October, in Korea.
+        <p class="display-1">"Hello! Sharing world"</p>
```
Comment (carlos-ssh) [inline index.html:76]: **Kindly provide the requested design by the template:**

_**Template: **_
Screen Shot 2021-11-19 at 6 19 30
Screen Shot 2021-11-19 at 6 18 50
Screen Shot 2021-11-19 at 6 19 05

#
_**- Yours:**_
- Fix your fonts and style:
Screen Shot 2021-11-19 at 6 21 39

- Fix image sizes:
Screen Shot 2021-11-19 at 6 25 02
- Align your footer items:
Screen Shot 2021-11-19 at 6 25 25


## Also kindly check your ``about.html``:

Screen Shot 2021-11-19 at 6 28 43

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @MeqdamAlqudah , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @MeqdamAlqudah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

Looks like the issue was resolved here, so I have nothing else to say other than best of luck with the rest of your projects!

Keep rocking 🚀!

**Pull request merged**